### DISCUSSION TITLE
Add support to .ini file to configure the mod.

### DISCUSSION THREAD

**Discussion by x1nf3ctAdusX** [gennariarmando/v-hud#110] [Ideas]
Add support to .ini file to configure the mod just as the original GTAV hud, so we can put the old hud combined with the new one.

I was trying to put the old hud comnied with the new one but I don't know if it's possible.

I just started to use this mod because of the new menu and loading screen

**Comment by gennariarmando** [comment]
Why would you combine both, what are you even trying to achieve?

**Reply by x1nf3ctAdusX** [reply]
Sorry for being late, I forgot this discussion and what I wanted to achieve hahaha, but I think I wanted to confiurate this mod and use the old config file on it
